AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -74,13 +224,40 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
     5) Enter/Exit game mode.
     6) Test IsHidden.
     7) Test IsVisible.
-    8) Add Camera entity.
-    9) Add Camera component to Camera entity
-    10) Set the Directional Light component property Shadow|Camera to the Camera entity.
-    11) Delete Directional Light entity.
-    12) UNDO deletion.
-    13) REDO deletion.
-    14) Look for errors and asserts.
+    8) Set Color to purple, then back to default.
+    9) Intensity mode set to Nit then back to Ev100.
+    10) Set Intensity value to 10 then back to default of 4.
+    11) Set Angular Diameter to 1.0 then back to default of 0.5.
+    12) Add Camera entity.
+    13) Add Camera component to Camera entity.
+    14) Set the Directional Light component property Shadow|Camera to the Camera entity.
+    15) Shadow Far Clip set to 2000 then back to the default of 100.
+    16) Shadow Map size set to 256 then back to the default of 1024.
+    17) Cascade Count set to 2 then back to the default of 4.
+    18) Split ratio can be set to 0.3 then back to the default of 0.9.
+    19a) Turn Automatic Splitting off for subsequent tests.
```

Comment:
here and down in the test code itself: this counting appears incorrect? or at least its confusing to me. I would think 19b would come after 19a, not after 20.

Currently you have it numbered like this:
```
19a)
20)
19b)
21a)
22)
21b)
```

I would think it should be numbered this way:
```
19a)
19b)
20)
21a)
21b)
22)
```

or just remove the a's and b's altogether and just have more numbers: 19, 20, 21, 22, 23, 24, etc.

Reply:
The reason I used a and b is because they are part of the same property test. However because certain properties require others to be enabled/disabled in order to function, I frequently have to flip them in the right order. For example, I have to turn off "Automatic Splitting" (19a) to test "Far Depth Cascade" (20) then turn "Automatic Splitting" back on (19b).

I would prefer to leave them as is, because the a/b indicates that they are part of the same property test pair.

Reply:
I sent a DM about this to discuss further